CHANGES/251.bugfix

```diff
@@ -0,0 +1 @@
+Updated RPM auto-publish tests for future versions >=3.13.0.dev
```

Comment:
The changelog is meant for end users so test changes don't need to be called out. I would probably just repurpose this for the repository option as @mdellweg suggests.